Documentation/DebugMenu.md

```diff
@@ -13,15 +13,15 @@ Items are shown in the menu on a single line, so they use short codes and appear
 
 ### ID
 - This is used by Irons that have an ID and serial number to help check if the iron is authentic. All Pinecil V1 show the same ID number as this is the number programmed into the MCU.
-- The new Pinecil V2 released Aug. 2, 2022 now uses BL706, which enables generating a unique ID/Serial number to every iron. This can be used to verify your Pinecil authenticity [here](https://pinecil.pine64.org/).
+- The new Pinecil V2 released Aug. 2, 2022 now uses MCU BL706, which enables generating a unique ID/Serial number to every iron. This can be used to verify your [Pinecil authenticity here](https://pinecil.pine64.org/).
 
-### UpTime
+### Time

```

Comment:
```suggestion
### UpTime
```
Why would you change this, to just use the same term as description?
It can't be about space, since there is plenty.

'Time' in common sense refers to the actual time of the current day.

Reply:
it does not say Uptime in the  display.  it says Time in the menu. it is more accurate to the word on screen.

Reply:
Sorry about that, I forgot to change it in the menu screen as well. 😅